src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1057,7 +1057,7 @@ private bool IsValidFixedVariableInitializer(TypeSymbol declType, SourceLocalSym
 
             TypeSymbol elementType;
             bool hasErrors = false;
-            MethodSymbol getPinnableMethod = null;
+            MethodSymbol fixedpatternMethod = null;
```

Comment:
`fixedPatternMethod`

Reply:
fixed, thanks!

---
In reply to: [170078300](https://github.com/dotnet/roslyn/pull/24981#discussion_r170078300) [](ancestors = 170078300)